scilpy/tractanalysis/mrds_along_streamlines.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from scilpy.tractanalysis.grid_intersections import grid_intersections
+
+
+def mrds_metrics_along_streamlines(sft, mrds_pdds,
```

Comment:
I think that this method is not mrds-related. It's just dividing data.

It could be, in the main script: 
```
 mrds_sum, weights = \
        mrds_metric_sums_along_streamlines(sft, mrds_pdds,
                                           metrics, max_theta,
                                           length_weighting)
weighte_mrds = weight_values(mrds_sum, weights)
```
 
Where weight values is function that divides by the weight where non-zero, somewhere in volume tools or something.